src/Enum/OfficeType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle\Enum;
+
+use MabeEnum\Enum;
+
+class OfficeType extends Enum
+{
+    public const OFFICE = 'office';
+    public const APS = 'aps';
+
+    public static function office(): self
+    {
+        return static::get(static::OFFICE);
+    }
+
+    public static function aps(): self
```

Comment:
IIRC there also was an `mps` thing

Reply:
Hm, I think I misunderstood comment in task: AND-7663